### ISSUE TITLE
Add GSMA paper

### ISSUE THREAD

**Issue by benrito** [secretrobotron/research-website#1]
This is a good one and should perhaps be included:

https://stuff.webmaker.org/whitepapers/smartphones_content_skills.pdf

**Comment by secretrobotron** [comment]
@benrito yeah, cool. See how I featured mzl.la/globalweb on https://mozilla-foundation-research.herokuapp.com as an article? Should we feature this paper there as well separately, or try to add it to the globalweb page?

**Comment by benrito** [comment]
aha, I see!

what do you think?

want to show this in the research call?

**Comment by secretrobotron** [comment]
There isn't a lot of content on the blog atm. It would be easy to feature if you can write a tiny summary of it to put there.

It would be a bit of work to add it to the globalweb page, but we could do it. My recommendation: add it to the blog for now.

**Comment by benrito** [comment]
How’s this:

What is the baseline of digital literacy that is required for users to embrace the mobile internet, and to begin the journey toward becoming a “content creator”? What kinds of tools and user education programs could radically lower the barriers to content creation? Could investments in these kinds of training and tools have positive impactthe digital ecosystem in ‘mobile-first’ countries?
